docs/ext/smtliblexer.py

```diff
@@ -62,15 +62,16 @@ class SmtLibLexer(RegexLexer):
         '<', '>', '<=', '>=', '!=', '\+', '-', '\*', '/', 'div', 'mod', 'abs',
         'divisible', 'to_real', 'to_int', 'is_int', 'iand', 'int2bv',
         # separation logic
-        'emp', 'pto', 'sep', 'wand', 'nil',
+        'sep\.emp', 'pto', 'sep', 'wand', 'sep\.nil',
```

Comment:
Should we also rename `pto` and `wand`?

Reply:
`sep` would also be renamed to `sep.star` then.

Reply:
I guess we should do this.  This contradicts the syntax from https://sl-comp.github.io/docs/smtlib-sl.pdf although I'm not sure the syntax there should be considered stable.

Reply:
If we do this, I'll do this in a separate PR.

Reply:
Sounds good! I would be in favor of changing this.